src/Sylius/Bundle/UserBundle/Command/CreateUserCommand.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Core\Model\UserInterface;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Rbac\Model\RoleInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Aram Alipoor <aram.alipoor@gmail.com>
+ */
+class CreateUserCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('sylius:user:create')
+            ->setDescription('Creates a new user account.')
+            ->setDefinition(array(
+                new InputArgument('email', InputArgument::REQUIRED, 'Email'),
+                new InputArgument('password', InputArgument::REQUIRED, 'Password'),
+                new InputArgument('first-name', InputArgument::OPTIONAL, 'First name'),
+                new InputArgument('last-name', InputArgument::OPTIONAL, 'Last name'),
+                new InputArgument('currency', InputArgument::OPTIONAL, 'Currency'),
```

Comment:
UserBundle doesn't know about currencies.
CoreBundle know about them.
So it should be general command for UserBundle and extended version of CoreBundle, if it's required.